elephant/surrogates.py

```diff
@@ -0,0 +1,490 @@
+# -*- coding: utf-8 -*-
+"""
+docstring goes here.
+
+:copyright: Copyright 2014 by the Elephant team, see AUTHORS.txt.
```

Comment:
This is still to be decided. What I suggest is:

1) Firstly, everyone who contributes (above some threshold of contributions, so that just fixing a single typo doesn't count) puts their name and affiliation (if any) in AUTHOR.txt.
2) Secondly, when you contribute a module, class or function, you can put your name in the docstring if you wish. If someone subsequently modifies the code for that function, they can add their name, e.g. "Original implementation by X, performance improvements by Y."

I think it is simplest to keep the phrase "The Elephant team" in the copyright statement however; the actual copyright is probably held jointly by the institutions to which the contributors belong (FZJ, CNRS, etc.) although this depends on your contract (e.g. students in some countries get to keep the copyright).